packages/whale-api-client/src/api/stats.ts

```diff
@@ -53,7 +53,7 @@ export interface StatsData {
     address: number
     fee: number
     auction: number
-    payback: number
+    payback: string[]
```

Comment:
```suggestion
    payback: number
    paybacks: string[]
```

Surfacing an additional field so that we don't break backward compatibility since downstream already relies on it being a `number`, if it suddenly becomes a `string[]` will be a breaking change and cause critical systems that rely on it being a `number` to break.

> Since it's a deployed endpoint (`/v0/`), we need to be able to apply rolling updates and not cause breaking changes for consumer.